impeller/entity/entity_pass_target_unittests.cc

```diff
@@ -89,10 +83,7 @@ TEST_P(EntityPassTargetTest, SwapWithMSAAImplicitResolve) {
 
   auto entity_pass_target = EntityPassTarget(render_target, false, true);
 
-  auto color0 = entity_pass_target.GetRenderTarget()
-                    .GetColorAttachments()
-                    .find(0u)
```

Comment:
Can we instead change this to `.GetColorAttachment(0u)`.  That way the way the render target is storing the first color attachment isn't being leaked to its clients.

Reply:
Done